main/res/values/strings.xml

```diff
@@ -380,6 +380,7 @@
     <string name="caches_filter_missing_gcvote">Found caches not voted</string>
     
     <!-- caches lists -->
+    <string name="menu_lists">Manage lists &#8230;</string>
```

Comment:
As said before, I would leave out any symbol at this place. The title "Manage..." now speaks for itself and user will click it to see whats behind